Documentation/manualtest-guide.md

```diff
@@ -81,20 +60,21 @@ Requirements:
 
 * Requires a Windows machine running the Bridge
 * The Windows machine should be on a Windows domain 
-* A tool such as `psexec` is needed to start the Bridge as LOCAL SYSTEM. <br/>
+* A tool such as `psexec` is needed to start the Bridge as LOCAL SYSTEM. <br/> 
+  Download the tool from https://technet.microsoft.com/en-us/sysinternals/psexec.aspx <br />
   `psexec -s -h <command>` will start a command as an elevated LOCAL SYSTEM user
 
```

Comment:
This works well for me; the only condition (and I implicitly call it out here) is that you must specify the full path when executing startBridge.cmd - once you use psexec to start a new process under a new context, the working directory changes.